exalearn-ui/src/app/pages/login-page/login-page.component.ts

```diff
@@ -1,7 +1,6 @@
 import { Component } from '@angular/core';
 import { FormControl, FormGroup, Validators } from '@angular/forms';
 import { AuthService } from '../../services/auth.service';
-
 import { UserAuth } from '../../interfaces/interfaces';
 
```

Comment:
Usually, an empty line is used to separate the imports of built-in Angular modules and application modules.

Reply:
Fixed, i just used fix file :c